Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageCell.swift

```diff
@@ -27,13 +37,23 @@ public final class ChatMessageCell: _TableViewCell {
     
     override public func setUpLayout() {
         super.setUpLayout()
+
+        containerStackView.axis = .vertical
+        containerStackView.alignment = .center
```

Comment:
do you think we should set `messageContentView.width == contentView.width` constraint to make sure it fills all the space?

Reply:
Sorry, found the place where we set leading/trailing constraints 👍 I would actually go with width constraint as it's a more explicit way of doing this. But both ways work so leaving it's up to you 🙂

Reply:
The width constraint might have issues with RTL Languages? I think I prefer the current approach, it is probably safer 👍